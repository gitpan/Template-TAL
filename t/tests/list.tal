<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    <div tal:repeat="colour path:colours">
      #<span tal:replace="repeat/colour/index" />:
      <span tal:replace="colour">Colour of the rainbow</span>
      <span tal:condition="repeat/colour/even" tal:replace="string:(even!)" />
      <span tal:condition="repeat/colour/end" tal:replace="string:(end!)" />
    </div>
  </body>
</html>
