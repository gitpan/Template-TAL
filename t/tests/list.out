<html>
  <body>
    <div>#0: red (even!)</div>
    <div>#1: green</div>
    <div>#2: blue (even!) (end!)</div>
  </body>
</html>
